content/applications/inventory_and_mrp/quality/quality_reporting/quality_checks_report.rst

```diff
@@ -0,0 +1,181 @@
+
+Pie chart
+~~~~~~~~~
+
+To display quality check data in a pie chart, click the :icon:`fa-pie-chart` :guilabel:`(Pie Chart)`
+button in the top-left corner of the dashboard.
+
+The pie chart displays each quality check stage (:guilabel:`Failed`, :guilabel:`To do`, and
+:guilabel:`Passed`) as a slice of the pie, with the size corresponding to the percentage of quality
+checks assigned to that stage.
+
+Each slice of the pie appears in a unique color, and a key at the top of the chart denotes the stage
+represented by each color.
+
+To remove a stage from the pie chart, click its name in the key at the top of the chart. After doing
+so, the remaining stages expand to fill the space vacated by the removed stage. Click the stage name
+again to make the stage reappear on the chart.
+
+.. image:: quality_checks_report/pie-chart.png
+   :align: center
+   :alt: The pie chart graph view on the quality checks report dashboard.
+
+Pivot view
+----------
+
+To display quality check data in pivot view, click the :icon:`oi-view-pivot` :guilabel:`(Pivot)`
+button in the top-right corner of the dashboard.
+
+The pivot view displays data in the form of a spreadsheet.
+
+By default, each column is labeled with a date on which at least one quality check was passed or
+failed. The left-most column displays the earliest date, with each successive column displaying the
+next chronological date.
+
+By default, each row is labeled with the name of one of the three quality check stages:
+:guilabel:`Failed`, :guilabel:`To do`, and :guilabel:`Passed`.
+
+The cell at the intersection of a column and row displays a number, which signifies the number of
+quality checks that passed or failed on that specific date. Click on a cell to see the quality
+checks displayed in a list.
+
+To change the data displayed by the columns or rows, click the :guilabel:`Total` heading above
+either. Then, click it again to open a popover menu, from which a new variable can be selected.
+
+.. example::
+   Click the :guilabel:`Total` heading above the columns to remove the date data. Then, click the
+   heading again, and select :guilabel:`Product` from the popover menu. Each column is now titled
+   with the name of a product for which a quality check has been created.
+
+.. figure:: quality_checks_report/pivot-view.png
+   :align: center
+   :alt: The default pivot view on the quality checks report dashboard.
+
+   The default pivot view on the quality checks report dashboard.
+
+Kanban view
+-----------
+
+To display quality check data in a Kanban view, click the :icon:`oi-view-kanban`
+:guilabel:`(Kanban)` button in the top-right corner of the dashboard.
+
+Kanban view displays all existing quality checks in a Kanban board, with each check organized into
+one of three stages, depending on their status: :guilabel:`Failed`, :guilabel:`To do`, and
+:guilabel:`Passed`.
+
+Quality checks can be drag-and-dropped from one stage to another. Doing so changes the status of the
```

Comment:
```suggestion
Quality checks can be dragged-and-dropped from one stage to another. Doing so changes the status of the
```

Reply:
I would move "Doing so changes the status..." to an admonition block (important/warning). Most of the other actions users wouldn't have a direct impact on the rest of the app, so this is worth calling out